src/components/MapFollowLocation/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {useRef, useState} from 'react';
+import {FAB} from 'react-native-paper';
+import {
+  StyleURL,
+  UserLocation,
+  MapView,
+  UserTrackingMode,
+  Camera,
+} from '@rnmapbox/maps';
+import {Dimensions, StyleSheet} from 'react-native';
+import {useHeaderHeight} from '@react-navigation/elements';
+import {Portal} from 'react-native-paper';
+
+export default function MapWidget() {
+  const headerHeight = useHeaderHeight();
+  const windowHeight = Dimensions.get('window').height;
+  const [followUserMode, setFollowUserMode] = useState<UserTrackingMode | null>(
+    UserTrackingMode.FollowWithHeading,
```

Comment:
What other options are there for mode. The followWithHeading kinda makes the map rotate as the gyro rotates which might not be what you want. At least for me I prefer to have the make a bit more stable relative to the screen.

Reply:
Their is Follow, FollowWithHeading, FollowWithCoarse. I couldn't find any doc's what they mean exactly but we can experiment.